SimTracker/Common/interface/SiPixelChargeReweightingAlgorithm.h

```diff
@@ -143,51 +142,54 @@ inline SiPixelChargeReweightingAlgorithm::SiPixelChargeReweightingAlgorithm(cons
       UseReweighting(conf.getParameter<bool>("UseReweighting")),
       applyLateReweighting_(conf.exists("applyLateReweighting") ? conf.getParameter<bool>("applyLateReweighting")
                                                                 : false),
-      PrintClusters(conf.getParameter<bool>("PrintClusters")),
-      PrintTemplates(conf.getParameter<bool>("PrintTemplates")) {
+      PrintClusters(conf.exists("PrintClusters") ? conf.getParameter<bool>("PrintClusters") : false),
```

Comment:
this practice is discouraged. Time to add a `fillDescriptions`.

Reply:
I know, and I agree with you that it's time to add a `fillDescriptions`, I was even thinking about doing it... but I'm not the official maintainer of these codes... the digitizer (both Phase-1 and Phase-2) is lacking the `fillDescriptions` too. So maybe I can do it, but then I'd prefer to do it as a campaign for all these in a separate PR

Reply:
Since these switches are seemingly used for debug and hence should not change physics, I suppose you could make them `untracked` and provide default values. In a second round let's go with the `fillDescriptions`.

Reply:
ok I fixed these, and will do the `fillDescriptions` next time